src/noui.h

```diff
@@ -16,5 +16,7 @@ void noui_InitMessage(const std::string& message);
 
 /** Connect all bitcoind signal handlers */
 void noui_connect();
+/** Connect all bitcoind signal handlers to silent no-op functions */
+void noui_connect_noop_handlers();
```

Comment:
This should probably only be somewhere in `src/test`

Reply:
Good point! Fixed. Please re-review!